src/Boolean.ts

```diff
@@ -137,7 +136,7 @@ export const anyPass =
 export const both =
   <A>(f: Predicate<A>): Endomorphism<Predicate<A>> =>
   g =>
-    getFunctionSemigroup(SemigroupAll)<A>().concat(f, g)
+    allPass([f, g])
 
```

Comment:
or this one?
```ts
export const both = curry2T(anyPass)
```

Reply:
I'm still in the "probably uses point-free code too often" phase so I prefer the `curry2T` variant, but I'll leave it up to you, I'm happy either way.